client/state/concierge/info/reducer.js

```diff
@@ -0,0 +1,22 @@
+/** @format */
```

Comment:
The Concierge tooling will likely be expanded later, so the state object name `state.concierge.info` may end up a little too generic. Maybe rename to something like `state.concierge.signupForm`.

Reply:
Makes sense, updated.